src/core/lv2/Lv2Manager.cpp

```diff
@@ -146,10 +167,15 @@ Lv2Manager::Lv2Manager() :
 	m_supportedFeatureURIs.insert(LV2_URID__map);
 	m_supportedFeatureURIs.insert(LV2_URID__unmap);
 	m_supportedFeatureURIs.insert(LV2_OPTIONS__options);
+	m_supportedFeatureURIs.insert(LV2_WORKER__schedule);
 	// min/max is always passed in the options
 	m_supportedFeatureURIs.insert(LV2_BUF_SIZE__boundedBlockLength);
 	// block length is only changed initially in AudioEngine CTOR
 	m_supportedFeatureURIs.insert(LV2_BUF_SIZE__fixedBlockLength);
+	if (const auto fpp = Engine::audioEngine()->framesPerPeriod(); (fpp & (fpp - 1)) == 0)  // <=> ffp is power of 2 (for ffp > 0)
```

Comment:
Instead of:
`Engine::audioEngine()->framesPerPeriod()`
Maybe pull in the setting from ConfigManager again:
`ConfigManager::inst()->value( "audioengine", "framesperaudiobuffer" ).toInt()`
Or store it in some constant that isn't changed like `m_framesPerPeriod`?

Reply:
It would be doable, but I'm not sure what the discrepancy between the value from the savefile and `DEFAULT_BUFFER_SIZE` mean. I don't understand if the resetting to `DEFAULT_BUFFER_SIZE` is wanted - @jasp00 should we delete it?

Reply:
Right now, I think this issue is not related to this PR / not caused by this PR. It would affect anything else relying on the framerate. In order to get this PR going (and also in order to keep this PR small), I'd rather like to move this problem into a separate PR.

Reply:
I was going to suggest the same.